go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/fullstorydev/grpcurl v1.8.6
 	github.com/go-redis/redis v6.15.9+incompatible
 	github.com/go-sql-driver/mysql v1.6.0
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/glog v1.0.0
```

Comment:
There is a bunch of updates in `go.mod` that seem unrelated. Could you minimise the number of new/modified dependencies, or confirm that every change here is necessary?

Reply:
Yeah, I just ran go mod tidy, so guess it updated some other deps. I'll take a peeksie to see what's up. My gut feeling is that mayhaps it's the k8s deps. That glog looks pretty old though if I'm reading things correctly?

Reply:
I think we can upgrade `glog` in a separate PR, it looks reasonable.

Reply:
@vaikas What's your Go tooling version? It looks like it deletes some `// indirect` deps that our tooling did not, so maybe we just have different versions.

Reply:
Yeah, i am running with 1.18, so that may be it. IIRC there's a flag to go mod compatibilty. I'll look and retry.

Reply:
Done, I'll resolve this and if something is still amiss, please reopen and I'll fix.

Reply:
Ok. I'll cut a different PR from the head that _only_ does the go mod changes, correct?

Reply:
Thank you!